src/pages/workspaces/workspace/Data.js

```diff
@@ -406,18 +494,16 @@ class EntitiesContent extends Component {
     ])
   }
 
-  renderIgvButton() {
+  renderToolButton() {
     const { selectedEntities } = this.state
 
-    return h(Fragment, [
-      h(ButtonPrimary, {
-        style: { marginRight: '1rem' },
-        disabled: _.isEmpty(selectedEntities),
-        tooltip: 'Opens files of the selected data with IGV',
-        onClick: () => this.setState({ showIgvSelector: true })
-      }, [
-        'Open with IGV'
-      ])
+    return h(ButtonPrimary, {
+      style: { marginRight: '1rem' },
+      disabled: _.isEmpty(selectedEntities),
+      tooltip: 'Open files of the selected data',
```

Comment:
nit: The language about 'files of the selected data' feels specific to what IGV was doing. Should this just be 'Open the selected data'?